Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -152,19 +152,38 @@ public function configDataProvider(): array
                     'disable_default_promphp_metrics' => false,
                 ],
             ],
+            [
+                'Namesapce as dsn',
```

Comment:
```suggestion
                'Namespace as dsn',
```

Reply:
Hum... I should remove this test cause it has no sense. the string "%env(...)%" is never injected into the config array.